risc0/zkvm/prove/make-id/src/main.rs

```diff
@@ -12,12 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Creates RISC Zero MethodIDs from ELF files
+//!
+//! A simple command line tool for creating
+//! [MethodID](risc0_zkvm::host::MethodId)s from RISC-V ELF files. It is
+//! intended for basic, low-level use cases. In particular, users looking to
+//! create MethodIDs for Rust code are advised to instead use our zkVM: visit
+//! our [Risc Zero Rust Starter repository](https://github.com/risc0/risc0-rust-starter)
+//! for help getting started.
+
 use clap::Parser;
 use std::fs;
 
 use risc0_zkvm::host::{MethodId, DEFAULT_METHOD_ID_LIMIT};
 
-/// Generates a MethodID for a given RISC-V ELF binary.
+/// The command line arguments necessary to run [main].
```

Comment:
Note that this particular doc string gets used by `clap` to generate a nice `--help` message. So in this case, I think we should keep the previous text.